DEVELOPING.md

```diff
@@ -354,6 +354,36 @@ stacktrace that was saved as a txt file.
 $ ndk-stack -sym obj/arm64-v8a -dump trace.txt
 ```
 
+## Release management
+
+The Mapbox Maps SDK for Android makes stable releases every 6 weeks. Beta build is available in week 2 and a release candidate in week 4, which is promoted to stable in week 6.
+
+We release on a schedule to provide a level of predictability for our customers. Having a cadence that the team can rely on gives transparency and provides a rhythm that becomes second nature.
+
+### Why do we provide pre-releases?
+We provide pre-releases during development to enable the community and our customers to provide feedback.
+
+### What is the beta release?
+A beta release is a pre-release that exposes initial APIs, behaviors of upcoming features for early feedback, and the release branch is created. We make this release during week 2 of the release cadence.
+
+### What is the RC release?
+We finalize the scope of the beta release during week 4 of the cadence. We normally will not introduce any new APIs, and the RC is then promoted to stable in week 6.
```

Comment:
```suggestion
We normally will not introduce any new APIs and promote RC to stable in week 6.
```

Reply:
do not like such shortened version, will stick to mine, improved it a bit